vllm/platforms/cpu.py

```diff
@@ -68,8 +69,15 @@ def check_and_update_config(cls, vllm_config: VllmConfig) -> None:
 
         cache_config = vllm_config.cache_config
 
+        ipex_avaliable = find_spec("intel_extension_for_pytorch") is not None
+
         if cache_config and cache_config.block_size is None:
-            cache_config.block_size = 16
+            cache_config.block_size = 128 if ipex_avaliable else 16
+
+        if not ipex_avaliable and cache_config.block_size != 16:
+            raise RuntimeError(
+                f"--block-size={cache_config.block_size} requires"
+                " intel_extension_for_pytorch")
```

Comment:
This forces non-Intel CPUs to use block size 16

Reply:
Yes, they will use the custom paged attention only provides implementations with ```block_size=16```.

Reply:
Before this PR, CPU users were still able to configure the block size. Does this mean that if they were to choose a value that wasn't 16, they would see errors?

Reply:
If using IPEX, there will be no problem. Otherwise the custom kernel will raise a error.

Reply:
Ok, I've just verified this on my M4 Mac, setting it to 128 throws an error in `attention/ops/ipex_attn.py` (it's interesting that this is called IPEX attention rather than CPU attention since it's used for all CPU architectures, but that's a separate issue)

Reply:
Yes, they are sharing some interfaces.